### PR TITLE
Update spec names in prep for Chrome 56/Opera 43.

### PR DESCRIPTION
**Pull request by jpmedley**
https://www.chromestatus.com/features#milestone%3D56

### CODE REVIEW

File: macros/SpecName.ejs [hunk 1]
```diff
@@ -39,6 +39,10 @@ var lang = env.locale;
 // * Please use HTTPS links instead of HTTP links when possible.
 
 var specList = {
+    'AcceleratedShape': {
```
Comment (Elchi3) [inline macros/SpecName.ejs]: This is already in SpecName as "Shape Detection API". Please remove.

**Comment by jpmedley** [comment]
@Elchi3 Sorry for the late response on this. I've been out of the office. Can you please take another look.

**Comment by Elchi3** [comment]
mmhhh, using GitHub's UI for resolving the merge conflict wasn't the best idea :/

**Comment by Elchi3** [comment]
Although e62815e9947c720ca8784761c07ffe8f83c0c31c looks quite OK.

**Comment by Elchi3** [comment]
I cleaned up my mess in PR #91 which is now merged. Thanks for your changes!

**Pull request closed without merging**